enhancements/network/netflow.md

```diff
@@ -0,0 +1,107 @@
+---
+title: OpenShift Network Flow export and collection support
+authors:
+  - "@rcarrillocruz"
+reviewers:
+  - "@russellb"
+approvers:
+  - TBD
+creation-date: 2021-01-11
+last-updated: 2020-01-11
+status: provisional
+---
+
+# OpenShift NetFlow support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OVS (Open vSwitch) supports exporting network flow protocols like NetFlow, IPFIX and sflow.
+OpenShift uses OVN/OVS for OVN-Kubernetes network plugin. 
+OpenShift should have the ability to export cluster traffic using any of this protocols.
+
+## Motivation
+
+* Customers need to be able to get network flows out of OpenShift to be consumed by their collectors.
+* Customers need to be able to see a log of the network traffic that flows thru OpenShift cluster.
+* Customers need to get metrics and monitoring data of the OpenShift cluster network traffic to make decisions on capacity and/or security issues.
+
+### Goals
+
+- Add support for exporting all the network flows protocols supported by OVS from OpenShift.
+- Add an optional collector that consumes the network flows data and enriches it with Kubernetes metadata. 
```

Comment:
We should probably set a goal for what we want our optional collector to be able to do with this data.  

Maybe ... The first target use case for our optional collector is to dump flow data to logs.

and then later you clarify that log handling is out of scope for this enhancement.  I'm tempted to at least give cluster logging as an obvious optional solution here.

Reply:
Not quite follow you on this, what you mean by optional solution here.
I see cluster logging as an add-on, i.e. the optional collector dumps the flow data into stdout which then will be handled by cluster logging for aggregation, analytics, searching etc.
You mean suggesting that as a future use case?

Reply:
Resolving as we are re-scoping the proposal for just the flows export.